src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -58,7 +58,11 @@ public static function provideBytesAt(): array
     public function testUnwrap()
     {
         $expected = ['hello', 'world'];
-        $actual = UnicodeString::unwrap([new UnicodeString('hello'), new UnicodeString('world')]);
+
+        $s = static::createFromString('');
+
+        $actual = $s::unwrap(['hello', 'world']);
```

Comment:
`$actual = $s::unwrap([static::createFromString('hello'), static::createFromString('world')]);`

Reply:
yes it's done , sorry for my mistake . It's first PR here :)